docs/open-intro.md

```diff
@@ -13,14 +13,14 @@ Embedded optimization is of great importance in a wide range of engineering appl
 Relevant application domains include
 
 - **fast robotic systems** which are often controlled by optimization-based methodologies,
-- **autonomous vehicles** where the collision avoidance problem is stated as a nonconvex optimization problem,
+- **autonomous vehicles** where the collision avoidance problem is stated as a non-convex optimization problem,
```

Comment:
I think "nonconvex", similar to "nonnegative", is an acceptable spelling (see also [this post](https://english.stackexchange.com/questions/26837/is-the-use-of-a-hyphen-between-non-and-an-adjective-strictly-necessary))

Reply:
@regexident thank you very much for pointing out to these typos

Reply:
Fair, I've reverted `non-convex` back to `nonconvex`.